docs/.vuepress/config.js

```diff
@@ -268,13 +268,7 @@ module.exports = {
             'concepts/common-pitfalls'
           ]
         },
-        {
-          title: 'Prefect Idioms',
-          collapsable: true,
-          children: [
-            'idioms/idioms',
-          ]
-        },
+        '/core/idioms/idioms',
```

Comment:
Sorry for all the small reviews, but I think the location of this doesn't look very good.  Could we move this to right above "FAQ" for now?